### DISCUSSION TITLE
How to set youtube video autoplay

### DISCUSSION THREAD

**Discussion by jiesielota23** [fancyapps/ui#25] [Q&A]
Hi just want to ask for help, how could we enable autoplay with youtube videos or any videos to add to fancy box? Thank you!

**Comment by fancyapps** [comment]
Hi,

Check this post about Chrome's autoplay policies - https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

> web browsers are moving towards stricter autoplay policies in order to improve the user experience

Basically, there is no guarantee that the video will start playing automatically unless the sound is muted.

**Reply by jiesielota23** [reply]
Hello,

Thank you for your response, I really appreciate it. :)
